railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_dispatch.hsts_subdomains = true
```

Comment:
Ah okey `config.ssl_options` is different that `config.action_dispatch.ssl_options`. 

But you meant reusing `config.ssl_options` right?